.github/workflows/pr-automation.yml

```
@@ -73,13 +73,27 @@ jobs:
             const branch = context.payload.pull_request.base.ref
             
             if (milestones.data.length == 0) {
+              await github.rest.issues.createComment({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                body: '@' + context.payload.pull_request.merged_by.login +
+                ' There are no open milestones ... please create one'
```

We should make clear what is tried and what is needed to do. Maybe something like this (also in the exception of the throw section)

```suggestion
                ' The PR can't be associated to a milestone, because there is no open milestone. Please create a new milestone before merge.'
```